src/NHSD.BuyingCatalogue.Identity.UserDatabase/Deployment/CreateTestUsers.sql

```diff
@@ -11,7 +11,10 @@ BEGIN
     DECLARE @executiveAgencyRoleId AS nchar(5) = 'RO116';
 
     DECLARE @aliceOrganisationId AS uniqueidentifier = (SELECT TOP (1) OrganisationId FROM dbo.Organisations WHERE PrimaryRoleId = @ccgRoleId ORDER BY OdsCode);
+    DECLARE @aliceOrganisationName AS nvarchar(255) =  (SELECT TOP (1) Name FROM dbo.Organisations WHERE PrimaryRoleId = @ccgRoleId ORDER BY OdsCode);
```

Comment:
Please review this file, I can't see the variables `@aliceOrganisationName` & `@bobOrganisationName` used anywhere. I maybe missing something.